README.md

```diff
@@ -1 +1,136 @@
-# ToDoList
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+# 📖 [ToDoList] <a name="about-project"></a>
+
+**[ToDoList]** The ToDoList project is simple web application where a user can add and remove tasks they want to complete. The basic functionality  includes the ability to add a new task, mark a task as completed, and delete a task from the list.
+A typical implementation involve a form to enter a new task, a list of tasks with checkboxes to indicate completion status, and buttons to delete or clear all completed tasks. The list of tasks is stored in an array, and the application is dynamically update the display of tasks based on changes to the data.
+
+## 🛠 HTML and CSS <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  Combination of HTML and CSS
+
+### Key Features <a name="key-features"></a>
+
+- **[Add and Remove  Functionality]**
+- **[Responsive]**
+- **[Display list oftask]**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+use npm start command to run this app.
+### Prerequisites
+
+In order to run this project you need:
+
+1. Knowledge of HTML and CSS 
+2. VSCode or other text editor installed. [Link to Download VSCode](https://code.visualstudio.com/download)
+3. Node package. [Link to Download Node](https://nodejs.org/en/download)
+
+### Setup
+
+- Clone this repository to your desired folder:
+
+    Git clone:(https://github.com/binodbhusal/ToDoList)
+
+- Navigate to the location of the folder in your machine:
+
+    you@your-Pc-name:~$ cd Hello-Microverse
+
+### Install
+
+Install this project with:
+- No need to install this project just Open index.html in your Browser.
+
+
+### Usage
+To run the project, follow these instructions:
+
+  - After Cloning this repo to your local machine.
+  - Open the index.html in your browser.
+
```

Comment:
- [ ] This project uses webpack, it would be better if we list all the commands we need to run the project on a local machine, such as __npm install__, etc.